src/RawTransaction.php

```diff
@@ -1188,6 +1189,7 @@ public static function extract_input_signatures_p2sh($input, $message_hash, $scr
                     foreach ($script_info['public_keys'] as $key) {
                         if (self::_check_sig($sig, $message_hash, $key) == true) {
                             $signatures[$key] = $sig;
+                            break;
```

Comment:
err, afaik it doesn't break anything and otherwise we should make sure there are tests that fail xD

this changes makes it `O(n)` instead of `O(n^2)`, once a signature has been found for a pubKey there's not gonna be another signature for that same pubKey.